contracts/contracts/strategies/qf-maci/MACIQFBase.sol

```diff
@@ -476,6 +475,9 @@ abstract contract MACIQFBase is BaseStrategy, Multicall {
             }else if (recipientStatus == Status.Accepted) {
                 recipient.status = Status.InReview;
             }
+
+            recipient.lastUpdateAt = uint64(block.timestamp);
```

Comment:
Perhaps intentional, but worth noting that only setting this if the `recipientStatus != Status.None` means that it will not be set upon initial registration. This does still prevent frontrunning since recipients won't be reviewed if their status is `Status.None`, but it's important to be aware that valid registered recipients can have a `lastUpdateAt` of `0`.